server-common/src/main/java/org/apache/gravitino/server/authorization/MetadataIdConverter.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.gravitino.server.authorization;
+
+import org.apache.gravitino.MetadataObject;
+
+/** It is used to convert MetadataObject to MetadataId */
+public class MetadataIdConverter {
+
+  private MetadataIdConverter() {}
+
+  public static Long doConvert(MetadataObject metadataObject) {
+    // TODO Rely on MetaCache
```

Comment:
Throw an unsupportedException?

Reply:
If this method throws an UnsupportedException, It need to add the @DoNotCall annotation; otherwise, it will not pass CI. However, since this method is called in JCasbinAuthorizer, even if the @DoNotCall annotation is added, it still cannot pass CI."